secure_message/validation/domain.py

```diff
@@ -141,7 +149,15 @@ def validate_non_zero_field_length(self, field_name, length, max_field_len):
     def validate_field_length(field_name, length, max_field_len, data=None):
         if length > max_field_len:
             logger.info('Field is too large', field_name=field_name, length=length, max_field_len=max_field_len)
-            raise ValidationError(f'{field_name} field length must not be greater than {max_field_len}', field_name, [], data)
+            raise ValidationError(f'{field_name} field length must not be greater than {max_field_len}', field_name, [],
+                                  data)
+
+    @staticmethod
+    def validate_category_type(field_name, data=None):
+        if data in ['SURVEY', 'TECHNICAL', 'MISC']:
```

Comment:
Not sure if this is what you meant?

```suggestion
        if data not in ['SURVEY', 'TECHNICAL', 'MISC']:
```

Reply:
no actually I wanted to check if the provided data against category matches these three attributes.